pages/Stats.vue

```diff
@@ -0,0 +1,282 @@
+<template>
+  <div
+    id="stats-viewer"
+    class="mx-4 mt-4 flex h-[91vh] flex-col"
+  >
+    <div class="stats-header">
+      <h1 class="text-2xl font-bold text-gray-700">
+        Statistics
+      </h1>
+    </div>
+    <div class="flex items-center justify-between">
+      &nbsp;
+      <div class="font-semibold text-center">Total Examined Names: {{ numRecords }}</div>
+      <div class="flex items-center space-x-2">
+        <div class="checkbox-wrapper">
+          <input
+            id="stats-checkbox"
+            v-model="myStats"
+            type="checkbox"
+          >
+          &nbsp;<label for="stats-checkbox" class="font-bold" >My Stats</label>
+        </div>
```

Comment:
```suggestion
        <div class="checkbox-wrapper flex items-center space-x-1">
          <input
            id="stats-checkbox"
            class="h-4 w-4"
            v-model="myStats"
            type="checkbox"
            @change="fetchStats"
          >
          <label for="stats-checkbox" class="font-bold">My Stats</label>
        </div>
```
Adding the `@change` may be nice but it's not required. Other changes are aesthetic.

Reply:
I will not adding @change at the this release.

Reply:
Ok removed the `@change`
The other changes make the checkbox bigger and center aligned